src/sv-pipeline/scripts/format_svtk_vcf_for_gatk.py

```diff
@@ -69,162 +72,98 @@ def _parse_ploidy_table(path: Text) -> Dict[Text, Dict[Text, int]]:
     return ploidy_dict
 
 
-def create_header(header_in: pysam.VariantHeader,
-                  replace_ev_format: bool,
-                  remove_infos: Set[Text],
-                  remove_formats: Set[Text]) -> pysam.VariantHeader:
+def update_header(header: pysam.VariantHeader) -> None:
     """
     Ingests the given header, removes specified fields, and adds necessary fields.
 
     Parameters
     ----------
-    header_in: pysam.VariantHeader
-        input header
-    remove_infos: Set[Text]
-        set of info fields to remove
-    remove_formats: Set[Text]
-        set of format fields to remove
-
-    Returns
-    -------
     header: pysam.VariantHeader
-        gatk-style header
+        input header
     """
-    header = pysam.VariantHeader()
-    for sample in header_in.samples:
-        header.add_sample(sample)
-    for line in header_in.records:
-        # remove fields
-        if len(line.attrs) > 0 and 'ID' in line.keys() and (line['ID'] in remove_infos or line['ID'] in remove_formats):
-            continue
-        line_str = str(line)
-        # remove source line
-        if line_str.startswith('##source='):
-            continue
-        header.add_line(line_str)
-    # new fields
-    header.add_line('##INFO=<ID=END2,Number=1,Type=Integer,Description="Second breakend position">')
-    header.add_line('##INFO=<ID=OSVTYPE,Number=1,Type=String,Description="Original SVTYPE">')
-    header.add_line('##FORMAT=<ID=CN,Number=1,Type=Integer,Description="Copy number">')
     header.add_line('##FORMAT=<ID=ECN,Number=1,Type=Integer,Description="Expected copy number for ref genotype">')
-    if replace_ev_format:
-        header.add_line('##FORMAT=<ID=EV,Number=.,Type=String,Description="Classes of evidence supporting final '
-                        'genotype">')
-    return header
+    # Add these just in case (no effect if they exist)
+    header.add_line('##INFO=<ID=END2,Number=1,Type=Integer,Description="Second position">')
+    header.add_line('##INFO=<ID=CHR2,Number=1,Type=String,Description="Second contig">')
+
+
+def rescale_gq(record):
+    for sample in record.samples:
+        for gq_field in GQ_FIELDS:
+            if gq_field in record.samples[sample] and record.samples[sample][gq_field] is not None:
+                record.samples[sample][gq_field] = floor(record.samples[sample][gq_field] / 10)
 
 
 def convert(record: pysam.VariantRecord,
-            vcf_out: pysam.VariantFile,
-            remove_infos: Set[Text],
-            remove_formats: Set[Text],
             bnd_end_dict: Optional[Dict[Text, int]],
-            ploidy_dict: Dict[Text, Dict[Text, int]]) -> pysam.VariantRecord:
+            ploidy_dict: Dict[Text, Dict[Text, int]],
+            scale_down_gq: bool) -> pysam.VariantRecord:
     """
     Converts a record from svtk to gatk style. This includes updating all GT fields with proper ploidy, and adding
```

Comment:
It doesn't look like there are updates to GT in this script anymore

Reply:
Updated